CoreFoundation/PlugIn.subproj/CFBundle_Internal.h

```diff
@@ -30,20 +30,42 @@ CF_EXTERN_C_BEGIN
 #define PLATFORM_PATH_STYLE kCFURLPOSIXPathStyle
 #endif
 
-// FHS bundles are supported on the Swift and C runtimes, except on Windows.
+// Freestanding bundles are not supported on Darwin.
+#if !DEPLOYMENT_RUNTIME_OBJC
+#define FREESTANDING_BUNDLES 1
+#else
+#define FREESTANDING_BUNDLES 0
+#endif
+
+// FHS bundles are not supported on Darwin, Windows and Android.
```

Comment:
Incorrect: they're supported on Darwin if we build with the Swift runtime. See earlier comment.